packages/ssi-service/src/auth.js

```diff
@@ -111,9 +113,7 @@ class OpAuth {
    * Removes any garbage hashes from the page URL
    */
   _removeHashes () {
```

Comment:
If this function is not used anywhere and only a dead code inside it can you delete this function

Reply:
It is used inside the init function.

Reply:
Oh I see what you mean. I had commented the code inside the function for testing.
Pushing the uncommented line again.